### DISCUSSION TITLE
Duvidas sobre a área de front/ux/ui, outra facul, tecnólogo, cursos ou pós ?

### DISCUSSION THREAD

**Discussion by leocvlh** [frontendbr/forum#1492] [Discussão]
Oi Galera bom dia

Tenho 22 anos e acabei minha primeira graduação mas em outra área no caso arquitetura, e acabei descobrindo grande interesse na área de t.i no caso front/ux/ui/ designer, acabei me interessando pela área quando estava no 6/7 semestre da facul o que não compensaria desistir naquele ponto para começar outra, minha duvida no momento é fazer outra facul, alguma pós, cursos ou ate mesmo um tecnólogo voltados para área ? durante esse período de termino da facul acabei fazendo alguns cursos básicos na área  como logica de programação, html e css e uma aprofundada em ps e ai (são programas que já tenho certa experiencia por conta da facul), mas estou em duvida sobre o que fazer agora que quero mudar de área, se alguém conseguir dar algumas dicas sobre qual caminho seguir agr fiquei agradecido, vlw galera

**Comment by nathpaiva** [comment]
A melhor forma de você vir para a área de TI é começar a estudar por conta ou fazer cursos por fora, hoje não é necessário você fazer uma nova faculade.

Claro que é muito interessante porém você pode se inserir na área antes mesmo de fazer uma faculdade.

Caso você tenha condições de pagar cursos online te indico [o Alura](https://alura.com.br), que lá você tem trilhas tanto de front como de UX/UI.

Se você souber que quer trabalhar com `front-end` e você já sabe ingles, eu te indico [o Front-end Masters](https://frontendmasters.com/).

Além disso existe também a inciativa [do training center](https://github.com/training-center), onde a comunidade ajuda com mentorias e outras coisas.

Gosto também bastante de um post que o @felipefialho traduziu e ajuda muito a guiar na carreira de front-end, [Do Zero a Herói Front-End (Parte I)](https://www.felipefialho.com/blog/2016/do-zero-a-heroi-do-front-end-parte-1) e [Do Zero a Herói Front-End (Parte II)](https://www.felipefialho.com/blog/2016/do-zero-a-heroi-do-front-end-parte-2)

Acho que é isso 😁.

Só pra contextualizar, sou formada e Marketing e aos 24 anos comecei na carreira de desenvolvimento.

**Comment by mutheusalmeida** [comment]
Que legal, @nathpaiva! Eu estou cursando Administração e sou autodidata em design e desenvolvimento. Eu estudo desde 2017, e comecei com o artigo do Willian Justen [Como se tornar um desenvolvedor Front End](https://willianjusten.com.br/como-se-tornar-um-desenvolvedor-front-end/) e essa tradução do @felipefialho.

**Comment by andrelsa** [comment]
@leocvlh o meu caso foi parecido com o seu, antes de cursar Sistemas de Informação, iniciei Administração de Empresas e Matemática, só que não terminei nenhum curso. Fiz minha graduação na UFG de SI, acho que no seu caso, por ter uma formação, neste momento não vejo necessidade de iniciar um curso superior. Como citado faça cursos e estude por conta própria e vá montando o seu portfólio, participando de palestras, envolva-se com a comunidade para criar um network. Caso seja necessário um curso superior ou técnico, você poderia fazer algum curso EaD, já que o diploma seria apenas a sua necessidade. 

Enquanto isso, é criar uma rotina de estudos, não faltam cursos gratuitos e pagos, depende apenas de você, com dedicação, estudo e paciência você conseguirá alcançar o seu objetivo.

Boa sorte mano =D
